data/db_demo.yaml

```diff
@@ -12,6 +12,23 @@ user:
     roles:
       - View only
 
+streams:
+  - name: ZTF Public
+    altdata:
+      collection: ZTF_alerts
+      selector: [1]
+    =id: ztf_public
+  - name: ZTF Partnership
+    altdata:
+      collection: ZTF_alerts
+      selector: [1, 2]
```

Comment:
Why not `2`? Technically the "partnership stream" is everything with PID=2.

Reply:
That's why I called it a "selector". This is then used on K as in `{"candidate.programid": {"$in": <stream["altdata"]["selector"]>}}`, so that a filter defined on this stream will also get executed on `programid=1` data.